session.go

```diff
@@ -144,6 +144,10 @@ func NewSession(cfg ClusterConfig) (*Session, error) {
 		return nil, errors.New("Can't use both Authenticator and AuthProvider in cluster config.")
 	}
 
+	if cfg.SerialConsistency > 0 && !cfg.SerialConsistency.IsSerial() {
+		return nil, fmt.Errorf("the default SerialConsistency level is not allowed to be anything else but SERIAL or LOCAL_SERIAL. Recived value: %v", cfg.SerialConsistency)
+	}
```

Comment:
How about adding a test for this?

Reply:
And overall, how about adding another test for this new allowance usage with the two consistency levels?

Reply:
And overall, how about adding another test to verify that these two consistency levels are allowed for use?

Reply:
I have added some tests to cover each consistency level, please check it